x-pack/plugins/apm/e2e/cypress/support/step_definitions/rum/rum_dashboard.ts

```diff
@@ -61,7 +61,10 @@ Then(`should display chart legend`, () => {
   cy.get('kbnLoadingIndicator').should('not.be.visible');
   cy.get('.euiLoadingChart').should('not.be.visible');
 
-  cy.get(chartLegend).eq(0).invoke('text').snapshot();
+  cy.get(chartLegend, { timeout: DEFAULT_TIMEOUT })
```

Comment:
I think there is a default timeout already. Was the existing timeout too short?

Reply:
yes, it's value is probably low.